docs/src/domains/quick-start.md

```diff
@@ -2,34 +2,30 @@
 title: "Quick Start"
 weight: -70
 description: |
-  By default, a Platform.sh app is available at its Platform.sh domain. The following resources will help you take your application live in domain that you wish.
+  By default, a Platform.sh app is available at a domain generated by Platform.sh. The following resources will help you take your application live on your custom domain.
 ---
-
-
 {{< description >}}
 
-The custom domain allows you to use your unique branding and improves your web application recognition over the Internet.
-
+{{< note theme="warning">}}
+ Do not add a custom domain to your project until you are fully ready to change your DNS. Until that time, continue working with the Platform.sh generated URLS.  
+{{< /note >}}
 
-In this short section we will give you a very simple, typical example. More involved use-cases (such as site with many domains or multiple applications are simply variations on this).
 
-Suppose your project ID is `abc123` in the US region, and you've registered `mysite.com`.  You want `www.mysite.com` to be the "real" site and `mysite.com` to redirect to it.
+{{< note>}}
+Custom domains can only be added to the main environment on production plans (Standard or larger).
+{{< /note >}}
 
-## Configure `routes.yaml`
 
-First, configure your `routes.yaml` file like so:
+## Summary of steps
```

Comment:
The back to back notes above this section look like a little much to me. I'd recommend leaving the "warning" note where it is, and moving the regular "which plan can I configure a domain" note above the list in this section.